src/common/playlists/PlayHeaderActions.jsx

```diff
@@ -78,11 +80,21 @@ const PlayHeaderActions = ({ play }) => {
     }
   };
 
+  useEffect(() => {
+    (async () => {
+      const { data } = await axios.get(
+        `https://giscus.app/api/discussions?repo=reactplay/react-play&term=${play.path}`
+      );
+      setCommentCount(data.discussion.totalCommentCount);
```

Comment:
Does totalCommentCount will be 0 if no comments are available?

Reply:
yes

Reply:
@sudipkundu999 Anything on this issue?

Reply:
No @atapas, I tried a lot of things but the issue remains same, works fine by directly hitting the url but not from codesandbox/deployed link.
eg. [This](https://giscus.app/api/discussions?repo=reactplay/react-play&term=/plays/react-gradients) opens in browser but not in [codesandbox](https://codesandbox.io/s/giscus-api-7d4wl7?file=/src/App.js)

Reply:
All good!

Reply:
So @atapas @sudipkundu999 what is the fate of this issue?